api/v1alpha1/datadogagent_types.go

```diff
@@ -17,39 +17,41 @@ import (
 // DatadogFeatures are Features running on the Agent and Cluster Agent.
 // +k8s:openapi-gen=true
 type DatadogFeatures struct {
-	// OrchestratorExplorer configuration
+	// OrchestratorExplorer configuration.
 	OrchestratorExplorer *OrchestratorExplorerConfig `json:"orchestratorExplorer,omitempty"`
-	// KubeStateMetricsCore configuration
+	// KubeStateMetricsCore configuration.
 	KubeStateMetricsCore *KubeStateMetricsCore `json:"kubeStateMetricsCore,omitempty"`
-	// PrometheusScrape configuration
+	// PrometheusScrape configuration.
 	PrometheusScrape *PrometheusScrapeConfig `json:"prometheusScrape,omitempty"`
+	// Logs configuration.
+	LogCollection *LogSpec `json:"logCollection,omitempty"`
```

Comment:
nit suggestion: should we rename the struct type to `LogCollectionConfig` or something similar? feel free to disregard :)

Reply:
I decided to call this field `logCollection` because it represent a "feature",
and for most user it should be only

```yaml
features:
  logCollection:
    enabled: true
```

Reply:
Ah maybe you was talking about the struct name.
I can rename it.